tap_typeform/streams.py

```diff
@@ -136,6 +136,19 @@ def get_landings(atx, form_id, bookmark):
 
         yield from items
 
+def fetch_sub_questions(row):
+    '''This function fetches records for each sub_question in a question group and returns a list of fetched sub_questions'''
+    sub_questions = [] #Creating blank list to accommodate each sub-question's records
+
+    #Appending each sub-question to the list
+    for question in row['properties'].get('fields',[]):
```

Comment:
Are we plan to fetch these 3 fields only every time? If yes then can we define the fields in the schema? 

Instead of list use Yield, it is safer.

Reply:
1. Yes, we are fetching these 3 fields, so I think we can define them in the schema.
2.  No we can not. Actually, we are not iterating on the returned list, we use this list as the value of the key "sub_question" as shown in line [176](https://github.com/singer-io/tap-typeform/pull/62/files#:~:text=sub_questions%5B%27sub_questions%27%5D).

Reply:
Great, Lets do the First change.

Reply:
Changed schema for Questions stream as per suggestion.